ui/oopsyraidsy/data/07-dt/dungeon/yuweyawata.ts

```diff
@@ -0,0 +1,71 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export type Data = OopsyData;
```

Comment:
Should we add a TODO for the trash packs?

Reply:
I wasn't planning to add anything for the trash.  I can leave a TODO if you think it's worth it.